Deploy from foxy branch

tylerjw opened the pull request:
### Description

This is the first step in accomplishing this:

* https://github.com/ros-planning/moveit2_tutorials/issues/125

I already pushed this into the foxy branch, this just cleans up the main branch (so CI won't deploy the main branch any more).

vatanaksoytezer reviewed (changes requested):
We should add foxy branch to industrial CI as well: https://github.com/ros-planning/moveit2_tutorials/blob/main/.github/workflows/industrial_ci_action.yml

tylerjw commented:
> We should add foxy branch to industrial CI as well: https://github.com/ros-planning/moveit2_tutorials/blob/main/.github/workflows/industrial_ci_action.yml

I updated this to separate the CI runs for foxy and main.  I also updated the readme badges.
